src/utils/testing.js

```diff
@@ -283,6 +283,16 @@ function setMockObjects() {
                 composition: [],
                 location: "mine"
             },
+            conditionWidget: {
```

Comment:
wondering if this is correct place to add mock data (unless used by multiple plugins/components).
In this case I think mock data is specific to `conditionWidget`, I guess we can define it in conditionWidget spec file?